lib/Doctrine/Inflector/Rules/English/Uninflected.php

```diff
@@ -149,6 +151,8 @@ private static function getDefault(): iterable
         yield new Pattern('progress');
         yield new Pattern('rabies');
         yield new Pattern('rain');
+        yield new Pattern('recommended');
+        yield new Pattern('related');
```

Comment:
these two aren't nouns, why are they being pluralized somewhere?

Reply:
Laravel uses it, among other things, to determine the parameters name on a route, singularizing it, like: /images/{image}/, and table names for models, pluralizing it.
My guess is those are fairly common words to use in a URL, but to keep things consistent, maybe those should be handed there, and not here

Reply:
That be the way to go in my opinion. Also please add tests for other 2 words that are to stay :)